node/README.md

```diff
@@ -215,11 +215,26 @@ Both of these cases are handled by the electron-webpack-quick-start example.
 ### node-gyp and native dependencies
 
 Some node/electron versions are binary incompatible and require rebuilding of
-native node dependencies for electron. In offline mode, it may result in broken ABI.
-If you are seeing errors like
-`The module 'something.node' was compiled against a different Node.js version`,
-then pass `--electron-node-headers` option to flatpak-node-generator and set
-`npm_config_nodedir` to `flatpak-node/node-gyp/electron-current`.
+native node dependencies for electron. In offline mode, it may result in broken
+ABI. If you are seeing errors like `The module 'something.node' was compiled
+against a different Node.js version`, then pass `--electron-node-headers`
+option to flatpak-node-generator and set `npm_config_nodedir` to
+`flatpak-node/node-gyp/electron-current`.
+
+Note: If you're using `--xdg-layout`, then setting `npm_config_nodedir` should
```

Comment:
`--xdg-layout` is enabled by default since #289

Reply:
@gasinvein I'm heading for vacations soon and don't have time to update this PR (which has been open and approved since March), but you should have permissions to push to my PR branch. Feel free to take over and push the desired changes directly to my fork!

Reply:
Ok, edited.

Reply:
Thanks @gasinvein!